core/src/main/resources/hudson/tasks/ArtifactArchiver/help-artifacts.html

```diff
@@ -5,4 +5,27 @@
    -- except that <tt>","</tt> (<em>comma</em>) is the only supported separator.
    The base directory is <a href='ws/'>the workspace</a>.
    You can only archive files that are located in your workspace.
+</div>
+
+   <br>
+<div>
+   <p>Here are some examples of usage for pipeline: </p>
```

Comment:
```suggestion
   <p>Here are some examples of usage for Pipeline: </p>
```

Unsure about this one, but I think we generally capitalize it (pipeline being the concept, Pipeline the implementation in Jenkins).